README.md

```diff
@@ -159,6 +159,7 @@ To enable debug info, run your program with environment flags
 - **deleteTicket(id, callback)** - Delete a ticket by its id
 - **restoreTicket(id, callback)** - Restore a ticket by its id
 - **listAllTickets(filter, callback)** - List All Tickets, check list of [filters](http://developer.freshdesk.com/api/#list_all_tickets)
+- **filterTickets(filter, callback)** - Filter tickets, based on ticket [fields](https://developer.freshdesk.com/api/#filter_tickets)
```

Comment:
Please, add a **bold** notice about query format.
And please, mention that the attached URL already contains all info about this.